### PR TITLE
feat: add custom controller gains

### PR DESCRIPTION
**Pull request by lterrac**
Add Proportional and integral gain in `servicelevelagreements` CRD

### CODE REVIEW

File: config/crd/bases/systemautoscaler.polimi.it_servicelevelagreements.yaml [hunk 1]
```diff
@@ -14,7 +14,6 @@ spec:
     listKind: ServiceLevelAgreementList
     plural: servicelevelagreements
     singular: servicelevelagreement
-  preserveUnknownFields: false
```
Comment (DragonBanana) [inline config/crd/bases/systemautoscaler.polimi.it_servicelevelagreements.yaml:17]: we should add it back

File: config/crd/bases/systemautoscaler.polimi.it_podscales.yaml [hunk 2]
```diff
@@ -14,7 +14,6 @@ spec:
     listKind: PodScaleList
     plural: podscales
     singular: podscale
-  preserveUnknownFields: false
```
Comment (DragonBanana) [inline config/crd/bases/systemautoscaler.polimi.it_podscales.yaml:17]: we should add it back

**Review by DragonBanana — CHANGES_REQUESTED** [review]

**Review by DragonBanana — APPROVED** [review]
LGTM

File: config/crd/bases/systemautoscaler.polimi.it_podscales.yaml (hunk 2 above)
Comment (lterrac) [reply config/crd/bases/systemautoscaler.polimi.it_podscales.yaml:17]: Done

File: config/crd/bases/systemautoscaler.polimi.it_servicelevelagreements.yaml [hunk 3]
```diff
@@ -100,6 +100,10 @@ spec:
                 description: Proportional gain used by recommender. DC parameter
                 format: int32
                 type: integer
+              recommenderLogic:
```
Comment (DragonBanana) [inline config/crd/bases/systemautoscaler.polimi.it_servicelevelagreements.yaml:103]: This default value will not work

**Review by DragonBanana — CHANGES_REQUESTED** [review]
We should change the value to the new one with underscores or use the camel case one

File: config/crd/bases/systemautoscaler.polimi.it_servicelevelagreements.yaml (hunk 3 above)
Comment (lterrac) [reply config/crd/bases/systemautoscaler.polimi.it_servicelevelagreements.yaml:103]: solved

**Review by DragonBanana — APPROVED** [review]
LGTM

**Pull request merged**